tests/CMakeLists.txt

```diff
@@ -370,6 +371,14 @@ set_tests_properties(VersionCheck PROPERTIES PASS_REGULAR_EXPRESSION "Catch2 v${
 add_test(NAME LibIdentityTest COMMAND $<TARGET_FILE:SelfTest> --libidentify)
 set_tests_properties(LibIdentityTest PROPERTIES PASS_REGULAR_EXPRESSION "description:    A Catch2 test executable")
 
+add_test(NAME BazelEnvSeedTest
+  COMMAND $<TARGET_FILE:SelfTest> "[bazel-seed]"
+)
+set_tests_properties(BazelEnvSeedTest PROPERTIES
+  ENVIRONMENT "TEST_RANDOM_SEED=12345"
+  COST 1
+)
```

Comment:
There is no reason to specify COST manually.

Also, Bazel env vars are only meant to be used if the main Bazel env var is present to enable Bazel support.